config/initializers/routing_constants.rb

```diff
@@ -0,0 +1,2 @@
+FIREFOX_BROWSER_REGEX = /firefox/i
+INTEGER_ID_REGEX = /\d+/
```

Comment:
This is very generic regex which can be used by non-routing methods and classes as well
move this to a better place